### DISCUSSION TITLE
Why the community chart?

### DISCUSSION THREAD

**Discussion by braykov** [minio/minio#19544] [Q&A] [answered]
This is just a question.

I am looking into installing a single instance of MinIO and I have been reading documentation in this repo and the official website.
Now I see two ways of installing:
 - the all recommended - operator and tenant - which is very well explained, even with videos, etc.
 - the community chart - clearly states it is "not supported", but nothing specific as to why it was created in the first place

And I don't see a single piece of reasoning explaining the need of this community chart.
So what is its purpose then, if not supported and there is an official way of installing? Something needs to make sense here. Why are there no differences stated? Why does it not say something along the lines of "...if you need this and that, use the community chart..."?

I am confused. How am I to choose a method?
On top of all, there is the [bitnami chart](https://github.com/bitnami/charts/tree/main/bitnami/minio) too.

**Comment by harshavardhana** [comment, marked as answer]
The community chart is maintained by the community, so it's unsupported, meaning any bugs arising from it are not our responsibility to fix or maintain. We are not interested in promoting it either. 

We have to keep it in this repo because this is the official repo. Bitnami is just another packager; why they write their own beats me. We do not support Bitnami charts either since they were not built under our guidance.

**Reply by dminca** [reply]
The question was not about support, that's distinguished clearly.

There's no comparison table or something of the sorts that we, responsible users, would be able to easily check differences between the official & unofficial offering.

In addition, keeping an _unsupported_ offering in the _official_ organisation makes it more misleading 😉

And why have you marked this as an answer and didn't allow @braykov to decide whether or not this answers his question? Wasn't this the sole purpose of discussions feature on GitHub?

**Reply by harshavardhana** [reply]
For us at the end its all about supportability.

**Reply by braykov** [reply]
In my particular situation, I am not able to pay for the license. So, I fully understand that I can not request or rely on any support.

These were my assumptions (because there is nothing written about it):
- the operator way is for licensed users (only??). You develop, support and endorse it.
- the community chart is for unlicensed users, single instances, or for trial purposes.

I hope this makes my original question more relevant now.
